hybrid agent release (#16238)
newrelic/docs-website: src/content/docs/release-notes/mobile-release-notes/cordova-release-notes/cordova-agent-626.mdx

---
subject: Cordova agent
releaseDate: '2024-02-20'
version: 6.2.6
downloadLink: 'https://github.com/newrelic/newrelic-cordova-plugin'
---

> # New in this release
> - Fixed a bug in the fetch instrumentation where customer options were inadvertently removed when no headers were specified. Now, options will be preserved even when headers are absent.

Reviewer: ```suggestion
##  New in this release

- Fixed a bug in the fetch instrumentation where customer options were inadvertently removed when no headers were specified. Now, options will be preserved even when headers are absent.
```

> - Addressed an issue that resulted in app crashes when an invalid URL was encountered in the capacitor plugin. To mitigate this, a valid URL checker has been implemented to ensure that mobilerequest events are created only with valid URLs.

Reviewer: ```suggestion
- Addressed an issue that resulted in app crashes when an invalid URL was encountered in the capacitor plugin. To mitigate this, we implemented a valid URL checker to ensure that `mobilerequest` events are created only with valid URLs.
```
